docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -71,26 +138,27 @@ please refer to our [supported modules definition](https://www.silverstripe.org/
-*  Both the docblock and error message should contain the **target version** where the functionality is removed.
-   So, if you're committing the change to a *3.1* minor release, the target version will be *4.0*. 
-*  Deprecations should not be committed to patch releases
-*  Deprecations should only be committed to pre-release branches, ideally before they enter the "beta" phase.
-   If deprecations are introduced after this point, their target version needs to be increased by one.
-*  Make sure that the old deprecated function works by calling the new function - don't have duplicated code!
-*  The commit message should contain an `API` prefix (see ["commit message format"](code#commit-messages))
-*  Document the change in the [changelog](/changelogs) for the next release
-*  Deprecated APIs can be removed only after developers have had sufficient time to react to the changes. Hence,     deprecated APIs should be removed in MAJOR releases only. Between MAJOR releases, leave the code in place with    a deprecation warning. 
-*  Exceptions to the deprecation cycle are APIs that have been moved into their own module, and continue to work     with the new minor release. These changes can be performed in a single minor release without a deprecation        period.
-
-Here's an example for replacing `Director::isDev()` with a (theoretical) `Env::is_dev()`:
-
+* Add a `@deprecated` item to the docblock tag, with a `{@link <class>}` item pointing to the new API to use.
+* Update the deprecated code to throw a [Deprecation::notice()](api:SilverStripe\Dev\Deprecation::notice()) error.
+* Both the docblock and error message should contain the **target version** where the functionality is removed.
+  So, if you're committing the change to a *3.1* minor release, the target version will be *4.0*. 
+* Deprecations should not be committed to patch releases
+* Deprecations should only be committed to pre-release branches, ideally before they enter the "beta" phase.
+  If deprecations are introduced after this point, their target version needs to be increased by one.
+* Make sure that the old deprecated function works by calling the new function - don't have duplicated code!
+* The commit message should contain an `API` prefix (see ["commit message format"](code#commit-messages))
+* Document the change in the [changelog](/changelogs) for the next release
+* Deprecated APIs can be removed only after developers have had sufficient time to react to the changes. Hence,     deprecated APIs should be removed in MAJOR releases only. Between MAJOR releases, leave the code in place with    a deprecation warning. 
+* Exceptions to the deprecation cycle are APIs that have been moved into their own module, and continue to work with the new minor release. These changes can be performed in a single minor release without a deprecation period.
+* Add a `warnings` entry to the module's `.upgrade.yml` file so the
```

Comment:
That's the only new entry i nthis list.

Reply:
It'd be nice it we could automate the population of this file from `@deprecated` tags in PHPDoc (just commenting, not requesting anything here)